### ISSUE TITLE
Don't change users bash_profile

### ISSUE THREAD

**Issue by ScreamingDev** [JDGrimes/wpppb#23]
```
echo Adding definition of WP_TESTS_DIR var to ~/.bash_profile...
echo "export WP_TESTS_DIR='${wp_tests_dir}'" >> ~/.bash_profile
```

This is only needed for travis I guess.
I don't want it to append things to my bash profile everytime I scaffold a new project.

**Comment by JDGrimes** [comment]
You're right, this is probably something that isn't useful for everyone. I use a single WP instance to work on all of my plugins, so I just need one path defined globally. I guess adding the path to `.bash_profile` should be optional.

And maybe there is a better way of configuring the path other than a env var. I'm just following what core does there, I guess. Suggestions welcome.
